src/Softeq.NetKit.Chat.Data.Persistent.Sql/Repositories/NotificationRepository.cs

```diff
@@ -62,12 +86,56 @@ public async Task<IReadOnlyCollection<Notification>> GetMemberNotificationsWithM
         {
             using (var connection = _sqlConnectionFactory.CreateConnection())
             {
-                var sqlQuery = @"SELECT * 
-                                 FROM Notifications n
-                                 INNER JOIN Messages m ON n.MessageId = m.Id
-                                 INNER JOIN Members me ON m.OwnerId = me.Id
-                                 INNER JOIN Channels c ON n.ChannelId = c.Id
-                                 WHERE n.MemberId = @memberId";
+                var sqlQuery = $@"
+                    SELECT 
+                        n.{nameof(Notification.Id)}, 
+                        n.{nameof(Notification.IsRead)}, 
+                        n.{nameof(Notification.MessageId)}, 
+                        n.{nameof(Notification.ChannelId)}, 
+                        n.{nameof(Notification.MemberId)},
+                        m.{nameof(Message.Id)},
+                        m.{nameof(Message.Body)},
+                        m.{nameof(Message.Created)},
+                        m.{nameof(Message.ImageUrl)},
+                        m.{nameof(Message.Type)},
+                        m.{nameof(Message.ChannelId)},
+                        m.{nameof(Message.OwnerId)},
+                        m.{nameof(Message.Updated)},
+                        m.{nameof(Message.ForwardMessageId)},
+                        m.{nameof(Message.AccessibilityStatus)},
+                        me.{nameof(Member.Id)},
+                        me.{nameof(Member.Email)},
+                        me.{nameof(Member.IsAfk)},
+                        me.{nameof(Member.IsBanned)},
+                        me.{nameof(Member.LastActivity)},
+                        me.{nameof(Member.LastNudged)},
+                        me.{nameof(Member.Name)},
+                        me.{nameof(Member.PhotoName)},
+                        me.{nameof(Member.Role)},
+                        me.{nameof(Member.SaasUserId)},
+                        me.{nameof(Member.Status)},
+                        me.{nameof(Member.IsActive)},
```

Comment:
Add me.{nameof(Member.IsDeleted)}, after merging new version.

Reply:
We won't forget :)